### PR TITLE
Update docs reg. EFGS shut down (Addresses #514)

### LINKED ISSUE #514: Clarify that the user doesn't have to select any countries for the EFGS and describe the 3 modes
## Current Implementation
The last sentence in https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md#cross-border-interoperability reads:

>In order for the EFGS to function correctly, all users must specify their visited countries correctly (either manually or automatically).

The 3 modes of the EFGS, which are f.e described [here](https://news.sap.com/2020/10/eu-corona-gateway-live/) are not covered in the documentation at all.

## Suggested Enhancement
Clarify that the user does not have to select any countries (or similar steps in order for the EFGS to work). 
Also, I would suggest to cover the 3 modes of the EFGS inside of the documentation and also say which mode is used and why.

## Expected Benefits
Better understanding for everybody who reads the documentation.

---

## Related Issues
- https://github.com/corona-warn-app/cwa-wishlist/issues/344

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR updates the solution_architecture.md document reg. the EFGS shut down. It adds a note to the start of the respective paragraph.

---

**Fixes #514**

### CODE REVIEW

File: solution_architecture.md [hunk 1]
```diff
@@ -396,6 +396,8 @@ If the back end calls from the mobile applications cannot be spread as evenly as
 
 ## CROSS-BORDER INTEROPERABILITY
 
+**Important update on cross-border interoperability: The European Federation Gateway Service was shut down EU-wide in February 2023. There is no more exchange of diagnosis keys via the gateway.** The following paragraphs describe the implementation up to the date of the shutdown in Febraury 2023.
```
Comment (MikeMcC399) [inline solution_architecture.md:399]: "Febraury" is misspelled

Comment (Ein-Tim) [reply solution_architecture.md]: Fixed with [`6cb9294` (#946)](https://github.com/corona-warn-app/cwa-documentation/pull/946/commits/6cb9294449a3353efa0919955873a56d8e65151c).

**Review by MikeMcC399 — APPROVED** [review]
No further comments!

**Comment by MikeMcC399** [comment]
@Ein-Tim 

Do you want to continue with this PR? Nobody has picked it up and now it is documented in the FAQs anyway [What happens to the EFGS for cross-country risk determination? How long will this service continue to run?](https://www.coronawarn.app/en/faq/results/#ramp_down_efgs)

image

To be consistent you would need to add other ramp-down dates into the architecture document. It is probably better not to add any dates into the `solution_architecture.md` document though.

Edit: The references to `teleTAN` in the document are also obsolete, due to the shutdown of the TAN Hotline (see [TAN Procedure](https://www.coronawarn.app/en/faq/results/#tan_procedure)). This is not mentioned in the [solution_architecture.md](https://github.com/corona-warn-app/cwa-documentation/blob/main/solution_architecture.md). This is the reason I say it would be inconsistent to mention about the shutdown of the EFGS without mentioning the shutdown of the TAN Hotline.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Thank you for the reminder! 

I still think that this is a good addition and would like to ask @dsarkar / @larswmh / @brianebeling to take a look at this!

**Comment by GisoSchroederSAP** [comment]
One quick comment: teleTAN and TAN hotline are under control and responsibility of TSI; they have and claim (by legal means) a different "communication pattern" (I have no better phrase, sorry). I'll take your comment to the next possible meeting. Let's see what the outcome will be. At least, we are not entitled to communicate about these activities without permission.

**Comment by Ein-Tim** [comment]
@GisoSchroederSAP This PR does not include any changes reg. TANs or the teleTAN hotline, so I'm not sure what you mean, sorry!

**Comment by MikeMcC399** [comment]
I'm sorry if my comments caused confusion and I don't really understand https://github.com/corona-warn-app/cwa-documentation/pull/946#issuecomment-1493106057 from @GisoSchroederSAP.

My point is that the document [solution_architecture.md](https://github.com/corona-warn-app/cwa-documentation/blob/main/solution_architecture.md) is purely an architecture document which makes no reference to the introduction or withdrawal of any features, which is why I think that this PR should not be implemented.

On April 30, 2023 all of the features from the [solution_architecture.md](https://github.com/corona-warn-app/cwa-documentation/blob/main/solution_architecture.md) will in any case be withdrawn. This point is already covered by the pinned issue https://github.com/corona-warn-app/cwa-documentation/issues/948 from @larswmh.

**Comment by Ein-Tim** [comment]
@MikeMcC399 Okay, I understand your point, reg. that this is a pure architecture document and that it doesn't reflect when features are implemented or when they are shutdown. I will leave this PR open so that the team can consider it, but I think your point is valuable, and it is ok for me if this PR is not merged!

**Comment by Ein-Tim** [comment]
@GisoSchroederSAP Can we expect further feedback here?

**Comment by MikeMcC399** [comment]
@Ein-Tim 

I suggest you close this PR.

To follow your logic of wanting to document the shutdown of the EFGS, you would need to follow-up now due to the connection to test centres being deactivated on April 20, 2023 and again on May 1, 2023 and modify the document once again, citing all the other servers which have been shut down. Then to make the rest of the repository consistent, you would need to modify many of the other documents accordingly.

The https://github.com/corona-warn-app/.github/blob/main/profile/README.md document can be used to pass a general message concerning all repositories affected by the ramp-down. This could mention which repository contents are still valid, or repositories could be set to archive status. Much of the information in the various README files will no longer be accurate after ramp-down and project closure.

**Comment by Ein-Tim** [comment]
@MikeMcC399 I agree with you, but I will not close this PR without having received further feedback from the team on what's the plan.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

What plan are you looking for? What should the plan cover? It sounds like this should be a separate topic, since you already agreed that the PR should be closed.

**Comment by Ein-Tim** [comment]
I'd just like to receive feedback on how the team will handle document updates after the app's shutdown. I will consider opening a new issue for this, but if we could get feedback here, that's also fine for me.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

> I'd just like to receive feedback on how the team will handle document updates after the app's shutdown. I will consider opening a new issue for this, but if we could get feedback here, that's also fine for me.

Considering that there have been no developer updates in this repository for the last 6 months I would be surprised if that changed in the remaining weeks until the official end of development milestone on May 31, 2023. Of course, you are entitled to pose the question, although this is a much more general question than the single topic about the EFGS server.

**Comment by Ein-Tim** [comment]
I will follow up in a separate issue or raise this issue internally.

**Pull request closed without merging**